packages/atomic/cypress/integration/breadbrumb-manager.cypress.ts

```diff
@@ -0,0 +1,153 @@
+import {setUpPage} from '../utils/setupComponent';
```

Comment:
General comments on the PR, putting it here for thread:

- We can divide breadcrumb tests into separate files, since each type of facet displays breadcrumb a bit differently e.g. category-facet-breadcrumbs.cypress.ts
- The part where th path is displayed in the breadcrumb is already tested, we could move those tests into aforementioned files.
- You can point to the branch is this PR https://github.com/coveo/ui-kit/pull/743 which touches relatively the same files.
- Follow the newer practices (like the category facet tests)
    - Create assertions file with breadcrumb assertions, reuse existing ones if possible
    - Rework selectors like in category-facet-selectors.ts, don’t use aliases
    - Use only part if possible, don’t need to use extra precise selectors
    - Reuse the “actions” if possible or create new actions e.g. “selectChildValueAt” instead of the common facet util.

Reply:
Note that Lucille will rework the facet tests to be similar to the category facet tests, the breadcrumb test might have to depend on these changes.